executors/cmdexec/device.go

```diff
@@ -37,12 +37,18 @@ func (s *CmdExecutor) DeviceSetup(host, device, vgid string, destroy bool) (d *e
 
 	// Setup commands
 	commands := []string{}
+	d_alignment := "256K"
 
 	if destroy {
 		logger.Info("Data on device %v (host %v) will be destroyed", device, host)
 		commands = append(commands, fmt.Sprintf("wipefs --all %v", device))
 	}
-	commands = append(commands, fmt.Sprintf("pvcreate -qq --metadatasize=128M --dataalignment=256K '%v'", device))
+
+	if s.config.PVDataAlignment != "" {
+		d_alignment = s.config.PVDataAlignment
+	}
```

Comment:
This is a great start but I would like to see the defaulting of the value handled like the recent gluster cli timeout is handled, with a helper function that handles returning the configured value or a default. Thanks

Reply:
This can be a private function in this case though.

Reply:
@phlogistonjohn implemented in a similar way, PTAL.